connect/runtime/src/test/java/org/apache/kafka/connect/runtime/ErrorHandlingTaskTest.java

```diff
@@ -494,72 +502,29 @@ private void assertErrorHandlingMetricValue(String name, double expected) {
         assertEquals(expected, measured, 0.001d);
     }
 
-    private void expectInitializeTask() {
-        consumer.subscribe(EasyMock.eq(singletonList(TOPIC)), EasyMock.capture(rebalanceListener));
-        PowerMock.expectLastCall();
-
-        sinkTask.initialize(EasyMock.capture(sinkTaskContext));
-        PowerMock.expectLastCall();
-        sinkTask.start(TASK_PROPS);
-        PowerMock.expectLastCall();
-    }
-
-    private void expectTaskGetTopic(boolean anyTimes) {
```

Comment:
These has been handled in methods, where this was called .

Reply:
I don't see a corresponding `verify` check in the `testErrorHandlingInSinkTasks` case. Can we add one?

Reply:
Hi @C0urante ,

I am not sure how to convert this to mockito . Can you please help ?